pkg/antctl/raw/multicluster/deploy/deploy_helper.go

```diff
@@ -91,6 +92,9 @@ func createResources(cmd *cobra.Command, apiGroupResources []*restmapper.APIGrou
 
 		obj, gvk, err := yaml.NewDecodingSerializer(unstructured.UnstructuredJSONScheme).Decode(rawObj.Raw, nil, nil)
 		if err != nil {
+			if string(content) == "Not Found" {
```

Comment:
This check can be moved after the line 187 `content := string(b)`.

Reply:
and it should check `resp.StatusCode == 404`, which is the well known code for Not Found, instead of the content, which may vary for different servers.